lib/tapioca/loaders/loader.rb

```diff
@@ -160,7 +160,7 @@ def with_rails_application(&blk)
         Rails.app_class = Rails.application = rails_application
       end
 
-      #: -> Array[singleton(Rails::Engine)]
+      T::Sig::WithoutRuntime.sig { returns(T::Array[T.class_of(Rails::Engine)]) }
```

Comment:
Now that we re-execute the signatures, having a reference to `Rails` is a problem. For now I changed them back to a `T::Sig::WithoutRuntime`. We can add a `@without-runtime` annotation later.